samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52820.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&radio {
+	status = "okay";
+	/* This is an example number of antennas that may be available
+	 * on antenna matrix board.
+	 */
+	dfe-antenna-num = < 10 >;
+	/* This is an example switch pattern that will be used to set an
+	 * antenna for Tx PDU (period before start of Tx CTE).
+	 */
+	dfe-pdu-antenna = <0x1>;
```

Comment:
Shouldn't spacing for values inside of `< >` be consistent?

Reply:
Yes, it should. It is corrected. Thank you.